egs/babel/s5d/conf/mfcc_hires_bp.conf

```diff
@@ -0,0 +1,10 @@
+# config for high-resolution MFCC features, intended for neural network training.
+# Note: we keep all cepstra, so it has the same info as filterbank features,
+# but MFCC is more easily compressible (because less correlated) which is why
+# we prefer this method.
+--use-energy=false   # use average of log energy, not energy.
+--sample-frequency=8000 #  Switchboard is sampled at 8kHz
```

Comment:
This is an entirely different configuration. The difference is the low-pass and high-pass corresponding to telephone bandwidth. Actually the low-pass should have been 330 Hz.